pkg/client/results/reader_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"path/filepath"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: This alias isn't needed

Reply:
I know; my editor (vscode) seems to really try and force it though. It does it automatically on save. I've been meaning to take a look at why (I figure it is just using goimports; maybe its out of date or something?)

Reply:
So:
 - I do have the newest version
 - this is actually tested behavior and I think it results from the fact that the path there isnt the actual path of the package. See [this change](https://github.com/golang/tools/commit/864069cfd1b106326ae556b3ca5c1d6b94645987).
 - I _thought_ that was the right path but it is actually a symlink to the staging directory:
```
$ ls -lah ~/go/src/github.com/kubernetes/kubernetes/vendor/k8s.io/
... ...  api -> ../../staging/src/k8s.io/api
```

Reply:
Honestly, it's a vscode thing, @davecheney pointed it out to be on some other project PR's. There's nothing in golang that requires this.

Reply:
It isn't a vscode thing; you're the 2nd person to mention that though so I'd be curious why that rumor got started (maybe it used to be true and just isnt?). I just tested with goimports directly and confirmed that it adds the same prefix.

[I came back to this PR just to find my reference to this because it came up again, then your message here compelled me to manually test it.]